packages/hash/frontend/src/pages/[accountId]/[pageEntityId].page.tsx

```diff
@@ -44,17 +44,23 @@ export const getStaticPaths: GetStaticPaths<{ slug: string }> = async () => {
  * @todo Include blocks present in the document in this
  */
 export const getStaticProps: GetStaticProps = async () => {
-  const preloadedBlockMeta = await Promise.all(
-    preloadedComponentIds?.map((componentId) => fetchBlockMeta(componentId)) ??
-      [],
+  const fetchedBlocksMeta = await Promise.all(
+    preloadedComponentIds.map(fetchBlockMeta),
```

Comment:
`.map` provides more arguments than `fetchBlockMeta` expects, which may cause confusing issues down the line – it's better to just be explicit here 

```suggestion
    preloadedComponentIds.map((componentId) => fetchBlockMeta(componentId)),
```